libs/blocks/recommended-articles/recommended-articles.js

```diff
@@ -0,0 +1,53 @@
+import {
+  buildArticleCard,
+  getBlogArticle,
+  loadTaxonomy,
+} from '../article-feed/article-helpers.js';
+
+import { createTag } from '../../utils/utils.js';
+
+async function decorateRecommendedArticles(recommendedArticlesEl, paths) {
+  if (recommendedArticlesEl.classList.contains('small')) {
+    recommendedArticlesEl.parentNode.querySelectorAll('a').forEach((aEl) => {
+      aEl.classList.add('button');
+    });
+    recommendedArticlesEl.parentNode.classList.add('recommended-articles-small-content-wrapper');
+  } else {
+    recommendedArticlesEl.parentNode.classList.add('recommended-articles-content-wrapper');
+    const title = document.createElement('h3');
+    recommendedArticlesEl.prepend(title);
+  }
```

Comment:
Are we attempting to update the document model on these so we can use the other blocks? The header etc. content would fall into a text block? Im not sure how much sifting is considered in regard to the output but we could likely rethink some of this structure and use more standard approaches. LMK if we need more direction on the expectations here.